x-pack/plugins/observability_solution/entity_manager/server/lib/entities/find_entity_definition.ts

```diff
@@ -60,27 +64,40 @@ async function getEntityDefinitionState(
 ) {
   const historyIngestPipelineId = generateHistoryIngestPipelineId(definition);
   const latestIngestPipelineId = generateLatestIngestPipelineId(definition);
-  const [ingestPipelines, transforms] = await Promise.all([
-    esClient.ingest.getPipeline({
-      id: `${historyIngestPipelineId},${latestIngestPipelineId}`,
+  const transformIds = [
+    generateHistoryTransformId(definition),
+    generateLatestTransformId(definition),
+    ...(isBackfillEnabled(definition) ? [generateHistoryBackfillTransformId(definition)] : []),
+  ];
+  const [ingestPipelines, indexTemplatesInstalled, transforms] = await Promise.all([
```

Comment:
We should just attach these like we do state in something like:
```typescript
{
  ...
  {
    resources: {
      ingestPipelines: [ ... ],
      indexTemplates: [ ... ],
      transforms: {
        history: { ... },
        latest: { ... },
        backfill: { ... },
        stats: { ... }
      },
    },
  }
  ...
}
```
That will make it a lot easier to debug for SDH because we can just ask them to run `GET kbn:/internal/entities/definition` and it will return their definition, index templates, ingest pipelines, transforms, and stats.... I have a few other ideas where this would be extremely handy.

Reply:
This makes sense, I'll do that as a follow up since this change is functionally complete and has been opened for a while. Can you clarify what you mean by `stats` ?

Reply:
Yes... the `transform/_stats` response, I believe it will have all the transforms stats since you're querying by transform ids?

Reply:
created https://github.com/elastic/elastic-entity-model/issues/141